### PR TITLE
*: update docs for pre_split_regions

### PR DESCRIPTION
**Pull request by crazycs520**
### What is changed, added or deleted? 
changed the doc of `split-region.md` and `create-table.md`

### What is the related PR or file link(s)? 

Related TiDB PR: https://github.com/pingcap/tidb/pull/11794

### Which version does your change affect? 

`dev`, `v3.0`, `v2.1`

### CODE REVIEW

**Comment by crazycs520** [comment]
@lilin90 PTAL

File: dev/reference/sql/statements/split-region.md [hunk 1]
```diff
@@ -174,7 +174,7 @@ region4  [("c", "")                    , maxIndexValue               )
 
 ## pre_split_regions
 
-使用带有 `shard_row_id_bits` 的表时，如果希望建表时就做均匀切分 Region，可以考虑配合 `pre_split_regions` 一起使用，用来在建表成功后就开始预均匀切分 `2^(pre_split_regions-1)` 个 Region。
+使用带有 `shard_row_id_bits` 的表时，如果希望建表时就做均匀切分 Region，可以考虑配合 `pre_split_regions` 一起使用，用来在建表成功后就开始预均匀切分 `2^(pre_split_regions)` 个 Region。
```
Comment (lilin90) [inline dev/reference/sql/statements/split-region.md]:
```suggestion
使用带有 `shard_row_id_bits` 的表时，如果希望建表时就均匀切分 Region，可以考虑配合 `pre_split_regions` 一起使用，用来在建表成功后就开始预均匀切分 `2^(pre_split_regions)` 个 Region。
```

File: dev/reference/sql/statements/create-table.md [hunk 2]
```diff
@@ -212,7 +212,7 @@ table_option:
 |----------------|--------------------------------------|----------------------------|
 |`AUTO_INCREMENT`|自增字段初始值                        |`AUTO_INCREMENT` = 5|
 |`SHARD_ROW_ID_BITS`|用来设置隐式 _tidb_rowid 的分片数量的 bit 位数 |`SHARD_ROW_ID_BITS` = 4|
-|`PRE_SPLIT_REGIONS`|用来在建表时预先均匀切分 2^(PRE_SPLIT_REGIONS-1) 个 Region |`PRE_SPLIT_REGIONS` = 4|
+|`PRE_SPLIT_REGIONS`|用来在建表时预先均匀切分 2^(PRE_SPLIT_REGIONS) 个 Region |`PRE_SPLIT_REGIONS` = 4|
```
Comment (lilin90) [inline dev/reference/sql/statements/create-table.md]:
```suggestion
|`PRE_SPLIT_REGIONS`|用来在建表时预先均匀切分 `2^(PRE_SPLIT_REGIONS)` 个 Region |`PRE_SPLIT_REGIONS` = 4|
```

File: v2.1/reference/sql/statements/create-table.md [hunk 3]
```diff
@@ -210,7 +210,7 @@ table_option:
 |----------------|--------------------------------------|----------------------------|
 |`AUTO_INCREMENT`|自增字段初始值                        |`AUTO_INCREMENT` = 5|
 |`SHARD_ROW_ID_BITS`|用来设置隐式 _tidb_rowid 的分片数量的 bit 位数 |`SHARD_ROW_ID_BITS` = 4|
-|`PRE_SPLIT_REGIONS`|用来在建表时预先均匀切分 2^(PRE_SPLIT_REGIONS-1) 个 Region |`PRE_SPLIT_REGIONS` = 4|
+|`PRE_SPLIT_REGIONS`|用来在建表时预先均匀切分 2^(PRE_SPLIT_REGIONS) 个 Region |`PRE_SPLIT_REGIONS` = 4|
```
Comment (lilin90) [inline v2.1/reference/sql/statements/create-table.md]:
```suggestion
|`PRE_SPLIT_REGIONS`|用来在建表时预先均匀切分 `2^(PRE_SPLIT_REGIONS)` 个 Region |`PRE_SPLIT_REGIONS` = 4|
```

File: v3.0/reference/sql/statements/create-table.md [hunk 4]
```diff
@@ -213,7 +213,7 @@ table_option:
 |----------------|--------------------------------------|----------------------------|
 |`AUTO_INCREMENT`|自增字段初始值                        |`AUTO_INCREMENT` = 5|
 |`SHARD_ROW_ID_BITS`|用来设置隐式 _tidb_rowid 的分片数量的 bit 位数 |`SHARD_ROW_ID_BITS` = 4|
-|`PRE_SPLIT_REGIONS`|用来在建表时预先均匀切分 2^(PRE_SPLIT_REGIONS-1) 个 Region |`PRE_SPLIT_REGIONS` = 4|
+|`PRE_SPLIT_REGIONS`|用来在建表时预先均匀切分 2^(PRE_SPLIT_REGIONS) 个 Region |`PRE_SPLIT_REGIONS` = 4|
```
Comment (lilin90) [inline v3.0/reference/sql/statements/create-table.md]:
```suggestion
|`PRE_SPLIT_REGIONS`|用来在建表时预先均匀切分 `2^(PRE_SPLIT_REGIONS)` 个 Region |`PRE_SPLIT_REGIONS` = 4|
```

File: v3.0/reference/sql/statements/split-region.md [hunk 5]
```diff
@@ -174,7 +174,7 @@ region4  [("c", "")                    , maxIndexValue               )
 
 ## pre_split_regions
 
-使用带有 `shard_row_id_bits` 的表时，如果希望建表时就做均匀切分 Region，可以考虑配合 `pre_split_regions` 一起使用，用来在建表成功后就开始预均匀切分 `2^(pre_split_regions-1)` 个 Region。
+使用带有 `shard_row_id_bits` 的表时，如果希望建表时就做均匀切分 Region，可以考虑配合 `pre_split_regions` 一起使用，用来在建表成功后就开始预均匀切分 `2^(pre_split_regions)` 个 Region。
```
Comment (lilin90) [inline v3.0/reference/sql/statements/split-region.md]:
```suggestion
使用带有 `shard_row_id_bits` 的表时，如果希望建表时就均匀切分 Region，可以考虑配合 `pre_split_regions` 一起使用，用来在建表成功后就开始预均匀切分 `2^(pre_split_regions)` 个 Region。
```

File: dev/reference/sql/statements/split-region.md [hunk 6]
```diff
@@ -185,10 +185,10 @@ region4  [("c", "")                    , maxIndexValue               )
 {{< copyable "sql" >}}
 
 ```sql
-create table t (a int, b int,index idx1(a)) shard_row_id_bits = 4 pre_split_regions=3;
+create table t (a int, b int,index idx1(a)) shard_row_id_bits = 4 pre_split_regions=2;
 ```
 
-该语句在建表后，会对这个表 t 预切分出 4 + 1 个 Region。4 (2^(3-1)) 个 Region 是用来存 table 的行数据的，1 个 Region 是用来存 idx1 索引的数据。
+该语句在建表后，会对这个表 t 预切分出 4 + 1 个 Region。4 (2^2) 个 Region 是用来存 table 的行数据的，1 个 Region 是用来存 idx1 索引的数据。
```
Comment (lilin90) [inline dev/reference/sql/statements/split-region.md]: Does the change in this file apply to `v2.1`? If yes, please update.

**Comment by lilin90** [comment]
@crazycs520 Please address comments. Thanks!

File: dev/reference/sql/statements/split-region.md (hunk 6 above)
Comment (crazycs520) [reply dev/reference/sql/statements/split-region.md]: done.

**Pull request merged**